brainwidemap/decoding/README.md

```diff
@@ -0,0 +1,120 @@
+# Pipeline for running BWM decoding analysis
```

Comment:
nice job @bensonbrandon, this looks great! very nice summary.

one comment - you can update the code snippets from
'''
sbatch stuff.py
'''
to 
```
sbatch stuff.py
```
by replacing the three apostrophes ''' with three backticks ``` and the markdown will render more nicely